src/Umbraco.Web.UI.Client/src/common/directives/components/upload/umbpropertyfileupload.directive.js

```diff
@@ -247,6 +232,7 @@
                 //special check for a comma in the name
                 newVal += files[i].name.split(',').join('-') + ",";
 
+                // TODO: I would love to remove this part. But I'm affright it would be breaking if removed. Its not used by File upload anymore as each preview handles the client-side data on their own.
                 if (isImage || extension === "svg") {
```

Comment:
Hi @bjarnef , this comment is here because the preview feature is moved into the media-preview of concern. In this case for SVG and Images this will from now on be handled by umbimagepreview. Which is a new thing by this PR.

Let me know if you see any breaking things in regards to that?